masterlist.yaml

```
@@ -9361,19 +9361,40 @@ plugins:
     inc: [ 'Marry Me Serana.esp' ]
     msg:
       - <<: *alsoUseX
-        subs: [ '[Serana Dialogue Edit](https://www.nexusmods.com/skyrimspecialedition/mods/16222/)' ]
-        condition: 'not active("Serana Dialogue Edit.esp")'
-      - <<: *patchProvided
-        subs: [ 'DX Crimson Blood Armor' ]
+        subs: [ 'Amorous Adventures Patch' ]
+        condition: 'active("SexLab-AmorousAdventures.esp") and not active("SDA-AA Patch.esp")'
+      - <<: *alsoUseX
+        subs: [ 'DX Crimson Blood Armor Patch' ]
         condition: 'active("Crimson Blood Armor.esp") and not active("SDA- DX CBlood Patch.esp")'
-      - <<: *patchProvided
-        subs: [ 'Lustmord Vampire Armor' ]
+      - <<: *alsoUseX
+        subs: [ 'Lustmord Vampire Armor Patch' ]
         condition: 'active("LustmordVampireArmor.esp") and not active("SDA-LustmordPatch.esp")'
+      - <<: *alsoUseX
+        subs: [ 'Nether''s Follower Framework Patch' ]
+        condition: 'active("nwsFollowerFramework.esp") and not active("SDA NFF Patch.esp")'
+      - <<: *patchProvided
+        subs: [ 'Relationship Dialogue Overhaul' ]
+        condition: 'active("Relationship Dialogue Overhaul.esp") and not active("SDA RDO Patch.esp")'
+      - <<: *patchProvided
+        subs: [ 'Serana Dialogue Edit' ]
+        condition: 'active("Serana Dialogue Edit.esp") and not active("SDA_SDE Patch.esp")'
+      - <<: *alsoUseX
+        subs: [ 'Wintersun - Faiths of Skyrim Patch' ]
+        condition: 'active("Wintersun - Faiths of Skyrim.esp") and not active("Complete Alchemy & Cooking Overhaul.esp") and not active("SDA Wintersun Patch.esp")'
+      - <<: *alsoUseX
+        subs: [ 'Wintersun and CACO Patch' ]
+        condition: 'active("Wintersun - Faiths of Skyrim.esp") and active("Complete Alchemy & Cooking Overhaul.esp") and not active("SDA Wintersun Patch.esp")'
     tag: [ Keywords ]
     clean:
       - crc: 0x61AFBCBD
         util: 'SSEEdit v4.0.3'
-
+    
+  - name: 'SDA Wintersun Patch.esp'
+    msg:
+      - <<: *useVersion
+        subs: [ 'Wintersun and CACAO Patch' ]
```

This fits the message a bit better:
```suggestion
        subs: [ 'Complete Alchemy & Cooking Overhaul' ]
```
Either that or:
```suggestion
        subs: [ 'CACO' ]
```
Whichever you prefer.